src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -144,6 +145,14 @@ class CromwellApiHandler(workflowManager: ActorRef) extends Actor {
         case Failure(ex: WorkflowManagerActor.WorkflowNotFoundException) => context.parent ! RequestComplete(StatusCodes.NotFound, ex.getMessage)
         case Failure(ex) => context.parent ! RequestComplete(StatusCodes.InternalServerError, ex.getMessage)
       }
+
+    case CallCaching(id, parameters, callName) =>
+      val eventualResponse = ask(workflowManager, WorkflowManagerActor.CallCaching(id, parameters, callName)).mapTo[Unit]
```

Comment:
Oh interesting, yeah this could pass the update count all the way through to the REST client.